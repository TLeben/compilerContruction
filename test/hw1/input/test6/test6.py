#!/usr/bin/python

input()
a1 = input()
print a1

print input()

a2 = input()
print a2

a3 = - input()
print a3

a4 = input() + 2
print a4

a5 = 2 + input()
print a5

a6 = 2 + - input()
print a6

a7 = - input() + 10
print a7

a8 = input() + input()
print a8

a9 = - input() + input()
print a9

