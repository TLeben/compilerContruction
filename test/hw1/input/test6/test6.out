10
100
1000
-10000
100002
1000002
-99998
-9990
1100
-9
